addon/components/o-s-s/star-rating.ts

```diff
@@ -1,11 +1,18 @@
 import Component from '@glimmer/component';
 import { assert } from '@ember/debug';
+import { action, set } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+type StarType = 'solid' | 'regular';
+type Star = { type: StarType };
 
 interface OSSStarRatingArgs {
   rating: number;
   totalStars: number;
   activeColor: StarColor;
   passiveColor: StarColor;
+  passiveStyle?: StarType;
+  onChange?: (rating: number) => void;
```

Comment:
```suggestion
  onChange?(rating: number): void;
```
This syntax is used instead